### ISSUE TITLE
Small indentation regression in 0.44.4

### ISSUE THREAD

**Issue by alexito4** [nicklockwood/SwiftFormat#570]
Hello! I was just updating swiftformat and found that `0.44.4` does an indentation change I'm not expecting. It's fine on `0.44.3`.

On 0.44.3:

```swift
        NotificationCenter.default.rx.notification(UIResponder.keyboardWillHideNotification)
            .asObservable()
            .subscribe { [weak self] _ in
                self?.keyboardHeight = nil
            }
            .disposed(by: self.bag)
```

On 0.44.4:

```swift
        NotificationCenter.default.rx.notification(UIResponder.keyboardWillHideNotification)
            .asObservable()
            .subscribe { [weak self] _ in
            self?.keyboardHeight = nil
        }
        .disposed(by: self.bag)
```

The diff:

```diff
         NotificationCenter.default.rx.notification(UIResponder.keyboardWillHideNotification)
             .asObservable()
             .subscribe { [weak self] _ in
-                self?.keyboardHeight = nil
-            }
-            .disposed(by: self.bag)
+            self?.keyboardHeight = nil
+        }
+        .disposed(by: self.bag)
```

<details><summary>Config:</summary>
<p>

```
# version 0.44.3
--allman false
--binarygrouping 4,8
--closingparen balanced
--commas always
--comments indent
--decimalgrouping 3,6
--elseposition same-line
--empty void
--exponentcase lowercase
--exponentgrouping disabled
--fractiongrouping disabled
--header ignore
--hexgrouping 4,8
--hexliteralcase uppercase
--ifdef indent
--importgrouping alphabetized
--indent 4
--indentcase false
--linebreaks lf
--octalgrouping 4,8
--operatorfunc spaced
--patternlet hoist
--self insert
--selfrequired 
--semicolons inline
--stripunusedargs always
--trimwhitespace always
--wraparguments preserve
--wrapcollections preserve

--disable andOperator,blankLinesAtStartOfScope,hoistPatternLet,numberFormatting,redundantRawValues,redundantReturn,sortedImports,typeSugar,unusedArguments
```
</p>
</details>

**Comment by nicklockwood** [comment]
@alexito4 I'm not able to reproduce this with that code sample and settings. Perhaps it's triggered by some other code in the file? Can you supply a sample file that I can use to reproduce the problem?

**Comment by alexito4** [comment]
Hi @nicklockwood , I've been able to reduce it to this example:

```swift
final class A {

    override func viewDidLoad() {
        if self.pinchToZoomIsEnabled {
            self.bla()
        }

        NotificationCenter.default.rx.notification(UIResponder.keyboardWillShowNotification)
            .asObservable()
            .subscribe { [weak self] event in
                self.bla = bla
            }
            .disposed(by: self.bag)

        NotificationCenter.default.rx.notification(UIResponder.keyboardWillHideNotification)
            .asObservable()
            .subscribe { [weak self] _ in
                self?.bla = bla
            }
            .disposed(by: self.bag)
    }
}
```

If you remove the `if` it doesn't happen. Or, if you remove the first notification observer, it doesn't happen. But when you have both the last lines are wrongly indented as shown in the original post.

I hope it helps!

**Comment by PompeiaPaetenari** [comment]
I believe this is similar to trailing closures having too much indentation of following lines?
```swift
  private func updateCamera() {
    let coordinateBounds =
      vm.markers.reduce(GMSCoordinateBounds()) { bounds, marker in
          bounds.includingCoordinate(marker.location)
        }
```
The first three lines are fine, but the last two have an extra amount of their indentation (we have it set to two spaces).  Currently, we disabled the indent rule, but several other locations in code end up with incorrect spacing and it is causing issues in our very massive codebase.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari this looks like a separate issue (it's not fixed by @zizi4n5's PR)

**Comment by nicklockwood** [comment]
@alexito4 @PompeiaPaetenari fixed in 0.44.5

**Comment by alexito4** [comment]
I can confirm that it works 👍  Thanks Nick!
